### ISSUE TITLE
Safari Browsers 

### ISSUE THREAD

**Issue by itsDZhang** [FranckFreiburger/vue-pdf#34]
I can view the pdf on chrome and firefox but not safari. I also tried safari on my iphone 6s (with latest IOS installed) and still no luck. The pdf just doesn't appear.

**Comment by FranckFreiburger** [comment]
Please try to open your file with http://mozilla.github.io/pdf.js/web/viewer.html?file=

**Comment by DZcoderX** [comment]
I tried that link and uploaded my pdf and it works. Does this mean I have to redirect the user to this link if they're using safari? (if so, how do I code it so the browser knows the user is using safari?)

**Comment by FranckFreiburger** [comment]
No, it just mean that is is not a pdf.js issue but a vue-pdf issue.

**Comment by DZcoderX** [comment]
Damn.. well hope fixing will be easy!

**Comment by leegoway** [comment]
I met the same issue on my android phone.

**Comment by FranckFreiburger** [comment]
Can you provide a console output ?

**Comment by FranckFreiburger** [comment]
Closed due to the lack of activity.

**Comment by iyerusad** [comment]
This still appears to repro on Safari and safari based devices (iOS). I am using this within a modal.

package.json: `vue-pdf": "^4.0.6`
My template code:
```
<template>
  <pdf :src="URL" />
</template>

<script>
import pdf from "vue-pdf";

export default {
  components: { pdf },
  props: {
    URL: {
      type: String,
      default: null
    }
  }
};
</script>
```

Testing in Safari 12.1 (latest as of posting), and latest iOS browser doesn't throw any exceptions. Just a blank spot where the PDF should be.

*Safari (bad) behavior:*
![Screen Shot 2019-04-18 at 4 03 28 PM](https://user-images.githubusercontent.com/25647368/56394194-ba2d7800-61f3-11e9-8904-b4f522d4d094.png)

*Chrome behavior (working properly):*
![chrome](https://user-images.githubusercontent.com/25647368/56394327-2e681b80-61f4-11e9-9bf2-795736961364.png)

**Comment by iyerusad** [comment]
Some more testing:
- Safari shows the same warning about `knockout groups not supported` (seems to imply PDF is loaded)
- Tried passing in a binary array instead of URL, same result (works in chrome, doesn't show in Safari)
- Tried different PDF, `https://cdn.filestackcontent.com/5qOCEpKzQldoRsVatUPS`. With exception of no longer seeing the `knockout groups not supported` (presumably this PDF doesn't have them), same result - no render in Safari and no error messages.
- Tested in Firefox, works similar to Chrome.

**Comment by Wanghattan** [comment]
Any update?
